### PR TITLE
Update user profile information

### PR DESCRIPTION
**Pull request by Kelechi194**
This is to update the user profile information taking into consideration the users has log into the portfolio site.
the Information to be goten from the database will be based on the user information stated in the login page. with that the user can directly access his profile information immediatly hi or she clicks update profile from the dash board.

### CODE REVIEW

**Comment by andrew21-mch** [comment]
can i see a screenshot

File: public/kelechi/uptain_session.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?php
+// Kowing that the user logs'-in, we automaticaly link to his session thus uptianing the user_id from it.
+session_start();
+require 'config.php'; 
+
+// Fetch the user's data
+$user_id = $_SESSION['user_id'];
+$query = $pdo->prepare("SELECT first_name, last_name, bio, image, phone FROM users WHERE id = ?");
+$query->execute([$user_id]);
+$user = $query->fetch(PDO::FETCH_ASSOC);
```
Comment (andrew21-mch) [inline public/kelechi/uptain_session.php:10]: why are you using pdo

File: public/kelechi/validate_user_input.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            die("Image size must not exceed 1MB.");
+        }
+        
+        // Move the uploaded file to a directory
+        $image = basename($image_file['name']);
+        move_uploaded_file($image_file['tmp_name'], "uploads/$image");
+    }
+    
+    // Update user data in the database
+    $query = $pdo->prepare("UPDATE users SET first_name = ?, last_name = ?, bio = ?, image = ?, phone = ? WHERE id = ?");
```
Comment (andrew21-mch) [inline public/kelechi/validate_user_input.php:33]: why are you using pdo

**Review by andrew21-mch — COMMENTED** [review]
don not use pdo

**Comment by Kelechi194** [comment]
new

**Pull request merged**